google-cloud-examples/src/main/java/com/google/cloud/examples/storage/buckets/RemoveBucketCors.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.examples.storage.buckets;
+
+// [START storage_remove_cors_configuration]
+import com.google.cloud.storage.Bucket;
+import com.google.cloud.storage.Cors;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import com.google.common.collect.ImmutableList;
+
+public class RemoveBucketCors {
+  public static void removeBucketCors(String projectId, String bucketName) {
+    // The ID of your GCP project
+    // String projectId = "your-project-id";
+
+    // The ID of your GCS bucket
+    // String bucketName = "your-unique-bucket-name";
+
+    Storage storage = StorageOptions.newBuilder().setProjectId(projectId).build().getService();
+    Bucket bucket = storage.get(bucketName);
+    bucket.toBuilder().setCors(ImmutableList.<Cors>of()).build().update();
```

Comment:
does null not work here?

Reply:
setting `null `is also works here

Reply:
@frankyn `null `setting doesn't works here see the respected example with response
```javascript

        Storage storage = StorageOptions.getDefaultInstance().getService();
        String bucketName = RemoteStorageHelper.generateBucketName();
        try{
            Cors cors = Cors.newBuilder()
                    .setOrigins(ImmutableList.of(Cors.Origin.of("*")))
                    .setMethods(ImmutableList.of(HttpMethod.GET))
                    .setResponseHeaders(ImmutableList.of("Content-Type"))
                    .setMaxAgeSeconds(100)
                    .build();
            Bucket bucket =
                    storage.create(BucketInfo.newBuilder(bucketName).setCors(ImmutableList.of(cors)).build());
            System.out.println("Prior size : " + bucket.getCors().size());
            Bucket updatedBucket  = bucket.toBuilder().setCors(null).build().update();
            System.out.println("After Updating operation size must be zero instead of : " + updatedBucket.getCors().size());

        }finally {
            RemoteStorageHelper.forceDelete(storage, bucketName, 5, TimeUnit.SECONDS);
        }
```
**Response :**
```response
Prior size : 1
After Updating operation size must be zero instead of : 1
```

Reply:
@frankyn I have raise the [issue ](https://github.com/googleapis/java-storage/issues/437)in `storage `repo regarding this and will fixed it soon

Reply:
I can see @frankyn 's argument, but following it's point, why need to call `setCors()` at all if it's empty or NULL?
Lots of folks (Effective Java 3rd ed.) prefer empty lists to using NULL, and I was going to comment w/ that, but really, if it's empty, why have to set it at all?

Reply:
In this case the example is to remove CORS from a bucket metadata. Good call, I think we can move forward with this PR with `ImmutableList.<Cors>Of()`

Reply:
Maybe one additional change that could be made here @athakor is to confirm that CORS are set before making a subsequent call. That way you reduce the need to make a patch request.

Reply:
done